7_react/react1/week2/todo_list_app/src/components/todoList.jsx

```diff
@@ -0,0 +1,79 @@
+import { useEffect } from "react";
+
+const TodoList = ({ todos, setTodos, setEditTodo }) => {
+  const handleComplete = ({ id }) => {
+    setTodos(
```

Comment:
Although it's not wrong but its better to use the parameter from `setter` directly, to ensure the data stay consists 
The same applies for all methods :) 
`  setTodos(prevTodos => prevTodos.map ... )`